examples/node_modules/package-json-redirects/README.md

```diff
@@ -33,4 +33,4 @@ Cons:
 
 - Lots of files and folders (aesthetic)
 - Need to add more every time a new subpath is added
-- Cannot support `*` wildcards
+- Need to create a `package.json` for all possible wildcard `*` matches 
```

Comment:
I’m ok with adding this demonstration, but I would leave this caveat reading a bit stronger. I guess creating a concrete target for every expected wildcard resolution is always a (tedious) alternative to using a wildcard, but being able to omit an exhaustive list of mappings is exactly the problem a wildcard is supposed to solve, so I wouldn’t exactly suggest that it’s a workaround for not having wildcards. I would still be convinced to go another route entirely if my `exports` used wildcards.

```suggestion
- Cannot support `*` wildcards; need to create a `package.json` for all possible matches
```

Reply:
Yes, absolutely agree! Just to contextualise, the scenario I imagine this strategy could be used is by a tool that essentially polyfills export wildcards. The tool would take a package.json#export containing wildcards, and produces a distributable containing the wildcard fallbacks. For the user, they would be spared the tedium of declaring multiple concrete exports, whilst still providing support to older targets. Later they may drop support for those targets without need for any migration to their code which is already using the latest standard.

Reply:
Btw if you don't feel this approach fits the purposes of this demonstration, no pressure to merge this. The PR can stand as documentation.